docs/UserGuide.md

```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
 ### Delete task: `delete` or `dl`
 Deletes the specified task from the planner. ATHENA will ensure to leave the index of other tasks untouched.
 
-#### Format: `delete INDEX`
+#### Usage
 
-* `INDEX` refers to the index number shown in the displayed task list. It must be a positive integer.
- 
-#### Example usage
+##### Command Format
 
-* `delete 1`
-* `dl 1`
+`delete TASK-ID`
+
+##### Parameters
+The `delete` command requires 1 parameter only.
+
+* `TASK-ID` refers to the number shown beside the task that the user wants to delete in the displayed task list. 
```

Comment:
maybe you could add a note here mentioning that the user can view the task number by using the "list" command